README.md

```diff
@@ -150,6 +152,10 @@ example in `rayon`.
 The [`bumpalo-herd`](https://crates.io/crates/bumpalo-herd) crate provides a pool of `Bump`
 allocators for use in such situations.
 
+### `allocator_api` support
+
+Bumpalo provides nightly' `allocator_api` support via the cargo feature `allocator_api`.
```

Comment:
Can you add more docs here briefly describing what the `allocator_api` nightly feature enables and provide a complete example that shows

* turning on the feature
* creating a `Bump`
* creates a `Vec` backed by the bump via the `new_in` method

Thanks